tools/yamcan/classes/Can.py

```diff
@@ -312,7 +312,81 @@ def calc_signal_params(self):
                 print(
                     f"Signal '{self.name}' uses discrete value table '{dv}' which has an invalid range"
                 )
-                self.is_valid = False
+            nat_rep.signedness = Signedness.unsigned
```

Comment:
All of this and below applies to discrete values because of the previous check, but none of it actually is applicable to discrete values. DVs dont have concepts of floats etc, theyre enums.

To be fair, this is a vlid thing todo, but you shouldnt have most of the following logic